featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -123,6 +123,46 @@ def test_make_trans_feat(es):
     assert v == 10
 
 
+def test_equal_categorical():
+    df = pd.DataFrame({
+        'id': range(4),
+        'value': pd.Categorical(['a', 'c', 'b', 'd']),
+        'value2': pd.Categorical(['a', 'b', 'a', 'd'])
+    })
+
+    es = ft.EntitySet('equal_test')
+    es.entity_from_dataframe('values', df, index='id')
+
+    f1 = ft.Feature([es['values']['value'], es['values']['value2']],
+                    primitive=Equal)
+
+    df = ft.calculate_feature_matrix(entityset=es, features=[f1])
+
+    assert set(es['values'].df['value'].cat.categories) != \
+        set(es['values'].df['value2'].cat.categories)
+    assert df['value = value2'].to_list() == [True, False, False, True]
+
+
+def test_not_equal_categorical():
+    df = pd.DataFrame({
+        'id': range(4),
+        'value': pd.Categorical(['a', 'c', 'b', 'd']),
+        'value2': pd.Categorical(['a', 'b', 'a', 'd'])
+    })
+
+    es = ft.EntitySet('not_equal_test')
+    es.entity_from_dataframe('values', df, index='id')
```

Comment:
can we make this entityset a fixture use by both tests to avoid duplicated code?